src/UTF8.cs

```diff
@@ -1277,7 +1277,18 @@ private unsafe static (int utfadjust, int scalaradjust) calculateErrorPathadjust
             }
             return GetPointerToFirstInvalidByteScalar(pInputBuffer + processedLength, inputLength - processedLength, out utf16CodeUnitCountAdjustment, out scalarCountAdjustment);
         }
-
+        public static void ToString(Vector128<byte> v)
+        {
+            Span<byte> b = stackalloc byte[16];
+            v.CopyTo(b);
+            Console.WriteLine(Convert.ToHexString(b));
```

Comment:
I think we can file an API proposal for dotnet/runtime to introduce something like:
```cs
var hex = v.ToString("X"); // common format symbol for hex
```

Reply:
Thanks!!! I did not mean to leave this code there. Removed.

I can file the proposal if you think that's useful that I do it, or I can support it if you do so (if that's useful that I do so).